### PR TITLE
l10n: Spelling unification

### PR DESCRIPTION
**Pull request by Valdnet**
Spelling unification in Transifex.

Signed-off-by: Valdnet <47037905+Valdnet@users.noreply.github.com>

### CODE REVIEW

File: templates/settings.admin.php [hunk 1]
```diff
@@ -620,7 +620,7 @@
 				<p class="note"><?php p($l->t(
 					'After you\'ve configured your webserver to enable shortened URLs, you should select the '
```
Comment (rakekniven) [inline templates/settings.admin.php:621]: Is "you've" ok?

File: templates/settings.admin.php [hunk 2]
```diff
@@ -620,7 +620,7 @@
...
 							. 'corresponding option above to let your users know about this feature. Don\'t enable '
```
Comment (rakekniven) [inline templates/settings.admin.php:622]: Is "Don't" ok?

**Review by rakekniven — COMMENTED** [review]
See my comments

**Review by rakekniven — APPROVED** [review]

**Comment by PhrozenByte** [comment]
Are there any other spelling unifications expected in the short term? There were a few complaints by translators in the past that they don't want to touch the large strings too often, so I want to make all changes simultaneously.

**Comment by Valdnet** [comment]
Nobody knows that. There are different ways to standardize text strings. Mostly when translating.

**Review by PhrozenByte — CHANGES_REQUESTED** [review]
Unlike server we don't attract that many translators. Merging such minor changes when expecting more in the future causes a huge setback in our goal having a fully translated app for more than just the "common" languages. Last time it took us months to recover...

Thus I'm not willing to approve this until I'm either sure that it's the only unification in the next couple of months, or it's a significant improvement. These are just "styling" changes that apply to English only. They don't have any effect on other languages - besides that we loose the translations for other languages. So it's not worth it **yet**. It's no doubt that this is a totally valid change and we'll merge it in the future, but we'll defer it until we have more significant changes.

Thank you for your improvements, looking forward to more spelling unifications :heart: :+1:

**Comment by Valdnet** [comment]
There is no good solution to this situation. We can only count on the support of translators 😉.

**Comment by PhrozenByte** [comment]
I agree :+1:

By the way, for changes like these it might be a good idea to also push the default `en` translation to Transifex, allowing us to do minor changes (like styling unifications, but also grammar or spelling errors that don't affect other languages) there instead of affecting all other languages. Such changes could be deferred until the next major release then. However, AFAIK this would require changes to server and how Transifex is incorporated by default. Do you know whether something like this has been discussed before? You think it might be worth creating an issue for this?

**Comment by Valdnet** [comment]
I know some developers do. Corrected text strings and new translations are only released in the new version.

> You think it might be worth creating an issue for this?

I think you can set up an issue 😉.

**Comment by rakekniven** [comment]
> Are there any other spelling unifications expected in the short term? 
No, currently there are no open issues at Transifex and only Valdnet or me are doing such changes like this PR.

> There were a few complaints by translators in the past that they don't want to touch the large strings too often, so I want to make all changes simultaneously.

That was me. That was and is more a usage problem. Transifex is still a pain to use with large strings.

> Do you know whether something like this has been discussed before?
Not that I know of. Currently NC is lacking resources for such changes.

File: templates/settings.admin.php [hunk 3]
```diff
@@ -565,25 +565,25 @@
 </article>
 
 <article class="section">
-	<h2><?php p($l->t('Configure your webserver')); ?></h2>
+	<h2><?php p($l->t('Configure your webs erver')); ?></h2>
```
Comment (rakekniven) [inline templates/settings.admin.php:568]: blank char at the wrong place

**Review by rakekniven — CHANGES_REQUESTED** [review]

**Review by rakekniven — APPROVED** [review]

File: templates/settings.admin.php [hunk 4]
```diff
@@ -565,25 +565,25 @@
...
 	)); ?></p>
 
 	<div class="message large">
 		<div class="icon icon-info"></div>
 		<div>
 			<p><?php p($l->t(
-				'Depending on your webserver\'s configuration, users can access their websites using different URLs. '
-						. 'By default, users can access their websites using Pico CMS for Nextcloud\'s full '
+				'Depending on your web server\'s configuration, users can access their websites using different URLs. '
+						. 'By default, users can access their web sites using Pico CMS for Nextcloud\'s full '
```
Comment (PhrozenByte) [inline templates/settings.admin.php:578]: I'd say that "web site" is a very uncommon notation, is it? Is this on purpose?

**Review by PhrozenByte — CHANGES_REQUESTED** [review]
I'm planning to release Pico CMS for Nextcloud 1.1 soon and thus changed the base branch to `cms_pico-1.1`. Please see the issue above and check whether there are more unifications necessary, I'll include them in v1.1 then. Thank you for your contributions :heart:

**Comment by rakekniven** [comment]
> I'm planning to release Pico CMS for Nextcloud 1.1 soon and thus changed the base branch to `cms_pico-1.1`. Please see the issue above and check whether there are more unifications necessary, I'll include them in v1.1 then. Thank you for your contributions ❤️

Currently there are no pending changes in our pipeline.
Feel free to merge and release.

**Pull request merged**